### DISCUSSION TITLE
Reset ekf after take-off with flow deck v2

### DISCUSSION THREAD

**Discussion by Kaijun101** [bitcraze/discussions#940] [Q&A] [answered]
I noticed that the flow deck drift when it is placed on the ground. There is normally some time before the crazyflie takes off and within this period, the drift could be significant. Is there any way, where I can reset the ekf to the origin when it has stabilised after take-off?

**Comment by krichardsson** [comment, marked as answer]
Yes, you can set the parameter [`kalman.resetEstimation`](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/params/#kalmanresetestimation) to a non-zero value to reset the kalman estimator. 
It will reset the position to the values of  the parameters `kalman.initialX`, 'kalman.initialY' and `kalman.initialZ`, (0, 0, 0) by default). 

You probably want to set `kalman.initialZ` to approximately the correct height before resetting to avoid too much confusion in the estimator and a smoother ride.

**Reply by Kaijun101** [reply]
That's great, thanks for your help!
